sklearn/multioutput.py

```diff
@@ -84,7 +89,7 @@ def _check(self):
 
 
 class _MultiOutputEstimator(MetaEstimatorMixin, BaseEstimator, metaclass=ABCMeta):
-    _parameter_constraints = {
+    _parameter_constraints: dict = {
```

Comment:
seems unrelated and unusual in scikit-learn.

Reply:
I'd agree, but this one is also not me here, it's from `main`:

https://github.com/scikit-learn/scikit-learn/blob/66a4d9639e27bc04c21bac4eb259fe28b1dad4af/sklearn/multioutput.py#L87

Reply:
`mypy` stuff